content/v2/domains/dnssec.markdown

```diff
@@ -0,0 +1,228 @@
+---
+title: DNSSEC API | Domains | DNSimple API v2
+excerpt: This page documents the DNSimple DNSSEC API v2.
+---
+
+# DNSSEC API
+
+* TOC
+{:toc}
+
+
+## Enable DNSSEC {#enable}
+
+    POST /:account/domains/:domain/dnssec
+
+Enable DNSSEC for the domain in the account. This will sign the zone. If the domain is registered it will also add the DS record to the corresponding registry.
+
+### Example
+
+Enable DNSSEC for the domain `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          -X POST \
+          https://api.dnsimple.com/v2/1010/domains/example.com/dnssec
+
+### Response
+
+Responds with HTTP 201.
+
+~~~json
+<%= pretty_print_fixture("/enableDnssec/success.http") %>
+~~~
+
+
+## Disable DNSSEC {#disable}
+
+    DELETE /:account/domains/:domain/dnssec
+
+### Example
+
+Disable DNSSEC for the domain `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          -X DELETE \
+          https://api.dnsimple.com/v2/1010/domains/example.com/dnssec
+
+### Response
+
+Responds with HTTP 204.
```

Comment:
I believe 412 is returned if the feature is gated, correct?

428 is the other response, when the feature is not enabled. I will add that.